library/aur.py

```diff
@@ -35,7 +35,6 @@
         description:
             - Whether or not to upgrade whole system.
         type: bool
-        default: no
```

Comment:
why are you removing this line? default of upgrade is `false`

Reply:
I did this to keep consistency with the parameter spec below. Regardless of the outcome of that thread, I can restore the default in the documentation. Let me know what you want to see here.

Reply:
I would leave the default in the documentation and the readme